v1/pkg/client/model_host.go

```diff
@@ -88,4 +88,6 @@ type Host struct {
 	AvailableFWBaselineID string `json:"AvailableFWBaselineID"`
 	// The version of the firmware baseline that is available for the host. This field is only populated when there is a new firmware baseline available.
 	AvailableFWBaselineVersion string `json:"AvailableFWBaselineVersion"`
+	// The serial number of the system that the host is running on.
+	SystemSerialNumber string `json:"SystemSerialNumber,omitempty"`
```

Comment:
I think GUI doesn't like the omitempty. That may be why all the other attributes are under "required" section.